Riot/Modules/Common/Buttons/RoundedButton.swift

```diff
@@ -64,6 +65,7 @@ final class RoundedButton: CustomRoundedButton, Themable {
         
         self.vc_setBackgroundColor(backgroundColor.withAlphaComponent(Constants.backgroundColorAlpha), for: .normal)
         self.setTitleColor(backgroundColor, for: .normal)
```

Comment:
Should you remove this line ?

Reply:
No, the first line set the background color.
The second line set the title color for **enabled** state of the button.
The third line set the title color for **disabled** state of the button (this is the addition).

Reply:
Oh you right. ;)